tutorial/part-1-creating-a-simple-view/creating-a-new-app.md

```diff
@@ -0,0 +1,28 @@
+# The Appsmith environment
+
+If this is your first time using Appsmith, you’ll need to get acquainted with some initial setup. When you log in, you'll be redirected to your Appsmith home-page. The home-page has an auto-generated organization called **Yourname apps** that consists of multiple templates. In this tutorial, we'll create our Catalog app in the same organization.
+
+{% hint style="info" %}
+Ensure that you use the auto-generated organization for the tutorial. It includes a mock database that you will need to follow the tutorial.
+{% endhint %}
+
+An app on Appsmith is a standalone software-application solving a specific use-case. An app can consist of one or more web-pages. Let’s create a new app Catalogue Dashboard in **Yourname apps**.
+
+1. Click on **Create New** under **Yourname apps**.
+2. You’ll be redirected to the configuration page of the new app.
```

Comment:
Because it's not really an app. It's taking to the editor window / page of that app. 
What do you think about - `You'll be taken to Appsmith editor of the app. This is where you configure your app.`

Reply:
Yes app editor is perfect. That's the term we use